src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -48,9 +54,45 @@ def get_request_by_action(self, action: str) -> SaaSRequest:
             )
             return request
         except KeyError:
-            raise ValueError(
+            logger.info(
                 f"The '{action}' action is not defined for the '{collection_name}' endpoint in {self.node.node.dataset.connection_key}"
             )
+            return None
+
+    def get_masking_request(self) -> Optional[SaaSRequest]:
```

Comment:
I moved this logic into the query config so we can update the SaaS query config's `action` field based on which request is chosen. The action is used for adding more context to logger statements and I didn't want it to be something that external code has to worry about setting.